lib/Backends/NNPI/NNPICompiledFunction.cpp

```diff
@@ -235,18 +253,32 @@ Error NNPICompiledFunction::compile(Function *F, const BackendOptions &opts) {
     return error;
   }
 
-  RETURN_IF_ERR(setupCompilationHints(F, newOpts.backendSpecificNodeInfo));
```

Comment:
This line looks like it was removed by mistake. (we should really stop merging every 5 min :) )
@jfix71 just keep this line as in master just above the "// Collect input/output names." comment.